cmd/goal/account.go

```diff
@@ -756,6 +759,7 @@ var listParticipationKeysCmd = &cobra.Command{
 var importCmd = &cobra.Command{
 	Use:   "import",
 	Short: "Import an account key from mnemonic",
+	Long:  "Import an account key from a mnemonic generated by the export command or by algokey (NOT a mnemonic from the goal wallet command). The imported account will be listed alongside your wallet-generated accounts, but will not be tied to your wallet mnemonic.",
```

Comment:
"will not be tied to your wallet mnemonic" sounds odd/misleading. Maybe just "will not be tied to your wallet" is better IMO.

Reply:
If you disagree that's fine, I don't have the strongest opinion; though we should probably revisit the language around mnemonics at some point (UX)